### ISSUE TITLE
NVDA should show a warning when installation of one or more add-ons cannot be completed

### ISSUE THREAD

**Issue by lukaszgo1** [nvaccess/nvda#12823]
### Is your feature request related to a problem? Please describe.
As described in #12629 in some cases it is impossible to finish installation of an add-on when upgrading because previous version of the add-on cannot be removed. While in such case log contains a warning explaining what went wrong this is not as discoverable as it could be.

### Describe the solution you'd like
NVDA should show a graphical warning when installation of one or more add-ons failed possibly explaining why it was impossible to install them - the more details the better.
### Describe alternatives you've considered
Leave code as is
### Additional context
Stems from discussion in #12792

**Comment by lukaszgo1** [comment]
FYI @CyrilleB79 I'm not going to work on this so if you're interested in having this implemented go ahead.
